README.md

```diff
@@ -271,6 +271,20 @@ ENV['ember-a11y-testing']: {
 }
 ```
 
+The color contrast check can cause undesired scrolling behavior.  To disable:
```

Comment:
The first example disables a rule and the other specifies an option for a check.  (As an aside, I'm still unclear on what the difference is between a check and a rule.)  In any case, I'm fine with however you want to proceed.  I submitted the PR because of how much hair-pulling was involved to get this figured out per #77, so I think it's justified to have examples in the readme.  Another idea could be to combine the examples (so there's only one `ENV['ember-a11y-testing'] = {` and possibly add comments for each directive.  Of course, then we'd have to change the rule that is disabled since we are specifying an option on the rule (check?) we just disabled...  Thoughts?